nimx/naketools.nim

```diff
@@ -182,9 +183,10 @@ proc newBuilder*(platform: string): Builder =
     let b = result
 
     b.platform = platform
-    b.appName = "MyGame"
-    b.bundleId = "com.mycompany.MyGame"
-    b.javaPackageId = "com.mycompany.MyGame"
+    b.appName = "NimxApp"
+    b.bundleId = "com.mycompany.NimxApp"
+    b.javaPackageId = "io.github.yglukhov.nimx"
```

Comment:
b.javaPackageId should be `com.mycompany.NimxApp` by default.
b.activityClassName should be `io.github.yglukhov.nimx.NimxActivity` by default.